plugins/woocommerce-blocks/tests/e2e/utils/shipping/shipping-utils.page.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { Page } from '@playwright/test';
+import { Admin } from '@wordpress/e2e-test-utils-playwright';
+
+export class ShippingUtils {
+	private page: Page;
+
+	constructor( page: Page ) {
+		this.page = page;
+	}
+
+	async openShippingSettings( { admin }: { admin: Admin } ) {
```

Comment:
Can we get the Admin passed in when constructing this class?

Reply:
See https://github.com/woocommerce/woocommerce/pull/44108#discussion_r1469451202